test/cypress/integration/pages/obtener-respuestas.js

```diff
@@ -0,0 +1,28 @@
+import { ObtenerRespuestasBuscar } from '../../pages/obtener-respuestas/buscar';
+
+const buscar = new ObtenerRespuestasBuscar();
+
+describe( 'obtener-respuestas', () => {
```

Comment:
Let's use the `beforeEach` thinger to avoid a little repetition:

```suggestion
describe( 'Obtener Respuestas', () => {
  beforeEach( () => {
    buscar.open();
  } );
```

And then remove each of the `buscar.open();` lines below.

Reply:
Done